test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
I think we need execute the callback in order to verify the code is valid, otherwise we'll get false positives.. unfortunately that requires us to stub parts of the request and response objects.